src/applications/disability-benefits/2346/sass/form-2346.scss

```diff
@@ -8,14 +8,14 @@
 @import "../../../../platform/forms/sass/m-form-confirmation";
 
 .order-background {
-  background-color: $color-gray-lightest;
+  background-color: #f1f1f1;
```

Comment:
If we can't use design utilities, we may want to indicate this to the design team to change their spec